### PR TITLE
Liquid cooler fixes

### LINKED ISSUE #18953: Paper cups don't work on water coolers
Using a paper cup on a water cooler just puts it back on the water cooler's "stack", but doesn't even do that. The cup just disappears, and you're left with no cool refreshing beverage.

Beakers and other reagent holders receive water just fine.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
del: You can no longer put paper cups back onto water coolers. In addition, they're now just called "liquid coolers" instead of changing based on their contents.
/:cl:
- Liquid coolers no longer change their names based on the liquid they hold ("That's a bacchus_blessing cooler.")
- You can no longer put paper cups onto liquid coolers. This is because of an unavoidable runtime that has to do with `afterattack` code.
- Paper cups' names have been lowercased.
- Closes #18953

### CODE REVIEW

**Comment by KazeEspada** [comment]
There was a reason it was paper and not paper cups.

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -160,10 +160,6 @@
 	tank_volume = 500
 	var/paper_cups = 25 //Paper cups left from the cooler
 
-/obj/structure/reagent_dispensers/water_cooler/New()
-	..()
-	name = "[reagent_id] cooler"
```
Comment (RemieRichards) [inline code/modules/reagents/reagent_dispenser.dm]:
```
var/datum/reagent/R = chemical_reagents_list[reagent_id]
name = "[R.name] cooler"
```

Comment (Xhuis) [inline code/modules/reagents/reagent_dispenser.dm]: The problem with that is that some reagents have their names the same as their IDs, plus their names are usually capitalized.

Comment (RemieRichards) [inline code/modules/reagents/reagent_dispenser.dm]: 's better than ```reagent_id```, which it should never have been given that's the non-ic reference to it.

**Pull request merged**